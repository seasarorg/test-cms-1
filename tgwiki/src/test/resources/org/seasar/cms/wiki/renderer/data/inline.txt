%%取り消し線%%
''強調''
'''斜体'''
((注釈))ここは注釈
someda@isenshi.com
http://www.isenshi.com
http://japan.cnet.com/news/ent/story/0,2000056022,20360651,00.htm
[[メール:someda@isenshi.com]]
[[twigaa:http://www.isenshi.com/]]
//WikiName siteConfig が null なのでテストは失敗する。

[[メール>someda@isenshi.com]]
[[URL>https://www.isenshi.com]]
[[CNET>http://japan.cnet.com/news/ent/story/0,2000056022,20360651,00.htm]]



改行~
~
~