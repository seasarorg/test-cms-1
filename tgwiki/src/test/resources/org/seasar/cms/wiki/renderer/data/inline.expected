<p><del>取り消し線</del><strong>強調</strong><em>斜体</em><a id="notetext_1" class="note_super" href="#notefoot_1">*1</a>ここは注釈<a href="mailto:someda@isenshi.com">someda@isenshi.com</a><a href="http://www.isenshi.com">http://www.isenshi.com</a><a href="http://japan.cnet.com/news/ent/story/0,2000056022,20360651,00.htm">http://japan.cnet.com/news/ent/story/0,2000056022,20360651,00.htm</a><a href="mailto:someda@isenshi.com">
メール
</a>
<a href="http://www.isenshi.com/">
twigaa
</a>
</p>
<p><a href="mailto:someda@isenshi.com">メール</a><a href="https://www.isenshi.com">URL</a><a href="http://japan.cnet.com/news/ent/story/0,2000056022,20360651,00.htm">CNET</a></p>
<p>改行<br/><br/><br/></p>
<hr class="note_hr" align="left"/>
<a id="notefoot_1" class="note_super" href="#notetext_1">*1</a>注釈<br/>
